api/api_ethereum.go

```diff
@@ -1132,77 +1213,127 @@ func (args *EthTransactionArgs) ToMessage(globalGasCap uint64, baseFee *big.Int,
 	data := args.data()
 
 	// TODO-Klaytn: Klaytn does not support accessList yet.
-	// var accessList AccessList
+	// var accessList types.AccessList
 	// if args.AccessList != nil {
 	//	 accessList = *args.AccessList
 	// }
 	return types.NewMessage(addr, args.To, 0, value, gas, gasPrice, data, false, intrinsicGas), nil
 }
 
+// toTransaction converts the arguments to a transaction.
+// This assumes that setDefaults has been called.
+func (args *EthTransactionArgs) toTransaction() *types.Transaction {
+	var tx *types.Transaction
+	switch {
+	case args.MaxFeePerGas != nil:
+		al := types.AccessList{}
+		if args.AccessList != nil {
+			al = *args.AccessList
+		}
+		tx = types.NewTx(&types.TxInternalDataDynamicFee{
+			ChainID:      (*big.Int)(args.ChainID),
+			AccountNonce: uint64(*args.Nonce),
+			GasTipCap:    (*big.Int)(args.MaxPriorityFeePerGas),
+			GasFeeCap:    (*big.Int)(args.MaxFeePerGas),
+			GasLimit:     uint64(*args.Gas),
+			Recipient:    args.To,
+			Amount:       (*big.Int)(args.Value),
+			Payload:      args.data(),
+			AccessList:   al,
+		})
+	case args.AccessList != nil:
+		tx = types.NewTx(&types.TxInternalDataAccessList{
+			ChainID:      (*big.Int)(args.ChainID),
+			AccountNonce: uint64(*args.Nonce),
+			Recipient:    args.To,
+			GasLimit:     uint64(*args.Gas),
+			Price:        (*big.Int)(args.GasPrice),
+			Amount:       (*big.Int)(args.Value),
+			Payload:      args.data(),
+			AccessList:   *args.AccessList,
+		})
+	default:
+		tx = types.NewTx(&types.TxInternalDataLegacy{
+			AccountNonce: uint64(*args.Nonce),
+			Price:        (*big.Int)(args.GasPrice),
+			GasLimit:     uint64(*args.Gas),
+			Recipient:    args.To,
+			Amount:       (*big.Int)(args.Value),
+			Payload:      args.data(),
+		})
+	}
+	return tx
+}
+
+func (args *EthTransactionArgs) ToTransaction() *types.Transaction {
```

Comment:
Why do we use two function to convert? ToTransaction -> toTransaction?

Reply:
Actually, this is from Ethereum.
In Ethereum, that functions are defined but as you said 
there are no places `ToTransaction` is used, so we can remove it if we want.

Reply:
okay. then no big deal! i will resolve this comment thanks.